From: Remi Collet <remi@remirepo.net>
Date: Thu, 30 Jul 2026 09:07:49 +0200
Subject: NEWS from 8.2.33

(cherry picked from commit 11b947ce14c7fcc77adc740349add4c5c8967a62)
(cherry picked from commit 9aa1f1c43308560bb5dc97cbf8995c66108aeef0)
(cherry picked from commit 78793e0bc0a244e228f9e9e9d2a28f48c941f223)
(cherry picked from commit f9e099c02fc0d8c37bd538eeaf48f4e0f3671d91)
---
 NEWS | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/NEWS b/NEWS
index ea6fc7f..62c276e 100644
--- a/NEWS
+++ b/NEWS
@@ -1,6 +1,19 @@
 PHP                                                                        NEWS
 |||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
 
+Backported from 8.2.33
+
+- GD:
+  . Upgrade libgd. (CVE-2026-9672) (Pierre Joye)
+
+- PGSQL:
+  . Fixed GHSA-7qpv-r5mr-78m4 (SQL injection via E'...' backslash breakout).
+    (CVE-2026-17543) (ilutov)
+
+- Phar:
+  . Fixed GHSA-vc5h-9ppw-p5f3 (Crash via recursive symlinks). (CVE-2026-7260)
+    (Jakub Zelenka)
+
 Backported from 8.2.32
 
 - OpenSSL:
